### ISSUE TITLE
GameFinder unsanitized paths crashing app due to failed debug assert

### ISSUE THREAD

**Issue by Al12rs** [Nexus-Mods/NexusMods.App#471]
There are still cases causing assert failures in xbox related code. 
GameFinder code needs fixing to sanitize all the paths.

**Comment by erri120** [comment]
This was blocked by #457, I'll update GameFinder this week.

**Comment by Al12rs** [comment]
If you have other stuff to do first I can try my hand at it, to unblock myself. Otherwise I can work from older commits on main for a bit.

**Comment by Al12rs** [comment]
Duplicate of #363
